apps/server/src/services/ElasticSearchClient.ts

```diff
@@ -48,24 +49,39 @@ export async function reIndexElasticSearch() {
   } catch {
     info('Elasticsearch "institutions" index did not exist')
   }
-  info('Elasticsearch indexing institutions')
```

Comment:
The name reIndex makes me think this is something that gets called more than once. Doesn't this only happen once?

Reply:
yeah I thought that too but didn't want to make this PR any bigger with more refactoring

Reply:
I don't think it would change the PR size much to make this stuff better. Adding functionality here and not making it all makes sense feels like adding tech debt